protocols/relay/src/behaviour/handler.rs

```diff
@@ -615,13 +611,8 @@ impl ConnectionHandler for Handler {
         }
     }
 
-    fn connection_keep_alive(&self) -> KeepAlive {
-        match self.idle_at {
-            Some(idle_at) if Instant::now().duration_since(idle_at) > Duration::from_secs(10) => {
-                KeepAlive::No
-            }
-            _ => KeepAlive::Yes,
-        }
+    fn connection_keep_alive(&self) -> bool {
+        !matches!(self.idle_at, Some(idle_at) if Instant::now().duration_since(idle_at) > Duration::from_secs(10))
```

Comment:
I love a good one-liner but this is a bit much :sweat_smile: 

Can we write this in multiple steps?

Reply:
If it helps, https://github.com/libp2p/rust-libp2p/pull/4692 should merge soon so you can use `let else` as well.

Reply:
fixed 3d10eb4